bin/offsetting/src/main.rs

```diff
@@ -20,6 +22,11 @@ enum Module {
   X360(X360Module),
   Oct(OctModule),
   Dct(DctModule),
+  #[command(name = "whynow")]
+  WhyNow(WhyNowModule),
+  Why(WhyModule),
+  #[command(name = "whyjustwhy")]
+  WhyJustWhy(WhyJustWhyModule),
```

Comment:
Maybe use the oppununaty of the rewrite to use more descriptive names? Backworts compatibility could still be posible.